src/backend/allocator/gbm.rs

```diff
@@ -0,0 +1,140 @@
+//! Module for Buffers created using [libgbm](reexports::gbm)
```

Comment:
It's pretty weird in the documentation to have this module existing while containing only the `GbmConvertError` enum. At least the documentation should explain that the user needs to directly use the `gbm::Device` type as its allocator, an maybe a few relevant types from `gbm` could be re-exported?